internal/log_analysis/log_processor/logtypes/logtesting/logtesting.go

```diff
@@ -37,8 +37,11 @@ import (
 	"github.com/panther-labs/panther/internal/log_analysis/log_processor/pantherlog/tcodec"
 )
 
+const ConfigAllowSkipEnvVar = "PANTHER_TESTING_LOG_ANALYSIS_LOG_TESTING_ALLOW_SKIP"
+
 // RunTestsFromYAML reads all test cases in a YAML file and runs them.
 func RunTestsFromYAML(t *testing.T, resolve logtypes.Finder, filename string) {
```

Comment:
```suggestion
// If filter are provided, only tests that match the filters are run 
func RunTestsFromYAML(t *testing.T, resolve logtypes.Finder, filename string, filter ...string) {
```

Maybe adding a `filters` vadiadic arg here would be easier.
Using `stringset.Contains` should be fine to filter the test cases.